### DISCUSSION TITLE
How to use solokey v2 PIV (SmartCard) funtionality

### DISCUSSION THREAD

**Discussion by provi1** [solokeys/solo2-cli#36] [Q&A] [answered]
Hi,
do any one knew how to use solokey v2 PIV (SmartCard) funtionality?
like create a certficate on the key or upload an existing certificate to the key.
and use the solokey then for SmardCard funktions like login to operating system.

**Comment by smlx** [comment, marked as answer]
PIV support is a WIP according to @nickray. See https://github.com/solokeys/solo2/discussions/88.
